### PR TITLE
Change config option to participant_type

### PR DESCRIPTION
**Pull request by TheBlocks**
The rlbot.cfg file now handles the types of participants like this:
```
participant_type_0 = human
participant_type_1 = rlbot
participant_type_2 = psyonix
participant_type_3 = possessed_human
```

### CODE REVIEW

**Comment by TheBlocks** [comment]
Fixed typo

**Comment by TheBlocks** [comment]
Need to add same functionality to GUI runner

**Comment by TheBlocks** [comment]
Fixed for runner_gui.py

File: RLBotFramework/gui/agent_frames/agent_frame.py [hunk 1]
```diff
@@ -68,17 +67,17 @@ def refresh_widgets(self):
         for widget in self.grid_slaves(row=3, column=1):
             widget.grid_forget()
         if self.player_type.get() == "Human":
-            self.is_bot.set(False)
-            self.rlbot_controlled.set(False)
+            self.participant_type.set("human")
```
Comment (dtracers) [inline RLBotFramework/gui/agent_frames/agent_frame.py]: this human and the Human above should be able to use the exact same value with a constant declared at the top of the file.

The same should go for the other bot types

**Comment by TheBlocks** [comment]
Updated to fit with requested change. Now uses a dict for lookup instead of if-else statements.

**Pull request closed without merging**